src/ast-types/mdast.ts

```diff
@@ -37,6 +38,12 @@ export interface IHTML extends Unist.Literal {
   type: 'html';
 }
 
+export interface IInlineHTML extends Unist.Parent {
+  type: 'inlineHtml';
+  attributes: Dictionary<string | true, string>;
+  tagName: string;
+}
```

Comment:
For the reason above, I'd say this should be treated as a literal, like `IHTML`

Reply:
Explained above.
There are two kind of html elements:
- inline
- block

Inline such as `span` or `a` can be placed among regular markdown markup, and markdown markup should be intereprted in such case. As demonstrated in the other coment, GH handles them as well.
Luckily, block elements do not need to be treated in the same way, so we do not need any parser nor anything.
I adjusted the code to account for that.